Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.11 AS builder-base
+
+# Get dependencies
+WORKDIR /go/src/github.com/nimbess/stargazer
+ENV GO111MODULE=on
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+RUN go mod verify
+
+# Build the binary
+FROM builder-base AS builder
```

Comment:
what's the point of doing this and copying everything again?

Reply:
This should be a copy if I understand the docker build correctly. It is an incremental build and is suggested as being better performance.